enterprise/wsproxy/wsproxysdk/client.go

```diff
@@ -0,0 +1,62 @@
+package wsproxysdk
```

Comment:
Thoughts on naming this `wsproxysdk.go` instead? Easier to find

Reply:
This matches the naming convention of codersdk

Reply:
I think we should change that too (albeit not in this change probs). `agentsdk` is called `agentsdk.go`

Reply:
Sure, I can change `wsproxysdk` here and we can change `codersdk` in another PR